WordPress/src/androidTest/java/org/wordpress/android/ui/accounts/signup/UsernameChangerFullScreenDialogFragmentTest.kt

```diff
@@ -33,6 +34,7 @@ import javax.inject.Provider
 /**
  * Test to verify that the UsernameChanger functionality is still intact.
  */
+@Ignore("Ignored because these type of Espresso tests can cause flakiness and overall slower completion time for the test suite.")
```

Comment:
I think we'll want to completely remove the tests before we merge this into develop. Having disabled tests which we are not planning to fix isn't probably worth it. I'll let @maxme decide as I'm not sure what he planned.

Reply:
Removed in https://github.com/wordpress-mobile/WordPress-Android/pull/10436/commits/a3f5c51c08a546eb35152e1ee1c1323a9c8d1cea 💯